Godeps/Godeps.json

```diff
@@ -10,6 +10,9 @@
 			"ImportPath": "github.com/antchfx/xpath",
 			"Rev": "1f3266e7730781347c9decd229c1fe1c18be760c"
 		},
+			"ImportPath": "github.com/dutchcoders/go-clamd",
```

Comment:
On one hand the existing project already supports both tcp and socket connections as well as a variety of existing scan methods that we'd may want to support anyway (or maybe not, see hand number 3).  On the other hand there isn't enough activity that would make me think it already has strong adoption.  On the third hand, as long as we define the interface in image-inspector on how we expect to interact with the scanner I don't care which dependency we compile in and what it does in the underlying implementation when we give it a path (walk and stream or filedes).

So, do we want to maintain a new version, enhance the existing one by adding a `ScanFileDes` to the [existing scan code](https://github.com/dutchcoders/go-clamd/blob/master/clamd.go), or fork?

Both projects are small enough.